awsutil/awsutil.go

```diff
@@ -45,6 +45,14 @@ func (e ECSTaskMeta) TaskID() string {
 	return split[len(split)-1]
 }
 
+func (e ECSTaskMeta) AccountID() (string, error) {
```

Comment:
It might be cleaner to use this https://github.com/aws/aws-sdk-go/blob/main/aws/arn/arn.go#L61-L76